attic/multibody/multibody_solvers.cc

```diff
@@ -648,8 +651,8 @@ void FixedPointSolver::AddJointLimitConstraint(const double tolerance) {
     // Ignoring the floating base coordinates if any.
     // We assume all floating base positions start with "base"
     if (tree_.get_position_name(i).substr(0, 4) != "base") {
-      prog_->AddConstraint(q_(i) >= (joint_min(i) + tolerance));
-      prog_->AddConstraint(q_(i) <= (joint_max(i) - tolerance));
+      prog_->AddBoundingBoxConstraint(joint_min(i) + tolerance,
```

Comment:
Assuming `tolerance` > 0, shouldn't this be 
prog_->AddBoundingBoxConstraint(joint_min(i) - tolerance, joint_max(i) + tolerance, q_(i)); ?

Reply:
`tolerance` is really a misnomer. this is more of a margin on the joint limit, so that the fixed point isn't right up against it. I'll rename.

Reply:
I see. Then probably the word `relaxation` here (https://github.com/DAIRLab/dairlib/blob/75c10731894aabb7e64d0bcb2e1f51694ef45fc6/attic/multibody/multibody_solvers.h#L173) can also be changed to `tightening` for example.